scripts/build/release.sh

```diff
@@ -40,7 +40,7 @@ Usage:
         The second argument indicates what type of release is it going to be released:
 
         - sprint: releases that are meant to be created each sprint end. A tag is automatically
-        generated along with the branch and develop is merged with master.
+        generated along with the branch.
```

Comment:
Note L47 below... "master is not updated" is not strictly correct... Have a look to https://github.com/telefonicaid/perseo-fe/pull/194/files#diff-db01dd9c2e5c291a004d2e3d3c73bff6L47